packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -210,11 +216,11 @@ function WorklistDataGrid(props) {
   // -------------------------------------
   // functions
 
-  const isRowSelectionCell = (columnIndex) => (
+  const isRowSelectionCell = useCallback((columnIndex) => (
     hasSelectableRows && columnIndex < displayedColumns.length && displayedColumns[columnIndex].id === WorklistDataGridUtils.ROW_SELECTION_COLUMN.id
-  );
+  ), [displayedColumns, hasSelectableRows]);
 
-  const setFocusedRowCol = (newRowIndex, newColIndex, makeActiveElement) => {
+  const setFocusedRowCol = useCallback((newRowIndex, newColIndex, makeActiveElement) => {
```

Comment:
We should not have added useCallback to this function.  There is not a need to memoize this function.

Reply:
Removed